core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -2770,28 +2785,27 @@ public function overrideAll()
 			{
 				foreach ($ids as $id)
 				{
+					// Initialize the active record
+					$this->loadActiveRecord($id);
```

Comment:
this seems really unperformant to query each record separately

Reply:
We can still improve that at some point. It's not a concern in this PR. It will only make it bigger.

Reply:
Did it anyway now in e95e7478fe02f10a0b59081914cdefb514259326.